tests/codeception/acceptance/BasicPageComponentsCest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Test the basic page components.
+ */
+class BasicPageComponentsCest {
+
+  /**
+   * Validate the Spacer Paragraph type exists
+   */
+  public function testSpacerParagraph(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/structure/paragraphs_type');
+    $I->canSee('Spacer');
+    $I->canSee("stanford_spacer");
+  }
```

Comment:
This is nice.

Reply:
i gotta set up javascript based tests still then we can get real fun